js/shared_controls.js

```diff
@@ -529,6 +541,7 @@ function Pokemon(pokeInfo) {
 		}
 		this.ability = (set.ability && typeof set.ability !== "undefined") ? set.ability :
 			(pokemon.ab && typeof pokemon.ab !== "undefined") ? pokemon.ab : "";
+		this.abilityOn = 	true;
```

Comment:
Too many spaces

Reply:
right, that was a tab i think which just looked like an extra space to me